client/src/pages/userview.js

```diff
@@ -28,7 +28,7 @@ const navigateToAddMeal = () => {
   navigate(`/api/nutrition`)
 }
 
-
+//unsure how to add id back in here need them for delete all need to to have as keys ??
 let meals = {};
```

Comment:
To make it a little easier to understand what you intend the structure of `meals` to be, you might want to do:

```
const meals = {
  Breakfast: [],
  Lunch: [],
  Dinner: [],
  Snacks: [],
};
```